traffic_ops/testing/api/v2/crconfig_test.go

```diff
@@ -153,6 +155,89 @@ func UpdateTestCRConfigSnapshot(t *testing.T) {
 	}
 }
 
+func MonitoringConfig(t *testing.T) {
+	if len(testData.CDNs) < 1 {
+		t.Fatalf("no cdn test data")
+	}
+	const cdnName = "cdn1"
+	const profileName = "EDGE1"
+	cdns, _, err := TOSession.GetCDNByName(cdnName)
+	if err != nil {
+		t.Fatalf("getting CDNs with name '%s': %s", cdnName, err.Error())
+	}
+	if len(cdns) < 1 {
+		t.Fatalf("expected to find a CDN named %s", cdnName)
+	}
+	if len(cdns) > 1 {
+		t.Fatalf("expected exactly 1 CDN named %s but found %d CDNs", cdnName, len(cdns))
+	}
+	profiles, _, err := TOSession.GetProfileByName(profileName)
+	if err != nil {
+		t.Fatalf("getting Profiles with name '%s': %s", profileName, err.Error())
+	}
+	if len(cdns) < 1 {
+		t.Fatalf("expected to find a Profile named %s", profileName)
+	}
+	if len(cdns) > 1 {
+		t.Fatalf("expected exactly 1 Profiles named %s but found %d Profiles", profileName, len(profiles))
+	}
```

Comment:
same as above RE: checking `cdns` instead of `profiles`

Reply:
Checking the length of `profiles` in 3963215ba4